euth/users/static/users/js/user_timezone.js

```diff
@@ -0,0 +1,6 @@
+var $ = window.jQuery
```

Comment:
You probably want to handle this type of renaming by using an [immediately invoked function expression](https://en.wikipedia.org/wiki/Immediately-invoked_function_expression) (iife). This is definitely not a server case, but what you are doing here is creating a global variable in the global namespace (called `$`). With an iife it would look like:
```
(function ($) {
   // your code here
})(window.jQuery)
```